packages/snaps-utils/src/handlers.ts

```diff
@@ -145,6 +153,19 @@ export const OnHomePageResponseStruct = union([
   OnHomePageResponseWithIdStruct,
 ]);
 
+export const OnSettingsPageResponseWithContentStruct = object({
+  content: ComponentOrElementStruct,
+});
+
+export const OnSettingsPageResponseWithIdStruct = object({
+  id: string(),
+});
+
+export const OnSettingsPageResponseStruct = union([
+  OnSettingsPageResponseWithContentStruct,
+  OnSettingsPageResponseWithIdStruct,
+]);
```

Comment:
```suggestion
export const OnSettingsPageResponseStruct = OnHomePageResponseStruct;
```

Wondering if we should just do this for now